.asf.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features
+
+github:
+  description: "Apache Accumulo Access Control Library"
+  homepage: https://accumulo.apache.org
+  labels:
+    - accumulo
+    - big-data
+    - hacktoberfest
+  features:
+    wiki: false
+    issues: true
+    projects: false
+
+notifications:
+  commits:      commits@accumulo.apache.org
+  issues:       notifications@accumulo.apache.org
+  pullrequests: notifications@accumulo.apache.org
+  jobs:         notifications@accumulo.apache.org
+  discussions:  notifications@accumulo.apache.org
```

Comment:
The job notifications could get a little spammy, and I don't think they are particularly useful in the mailing list archive. I would suggest dropping that. Unlike some of the others, I don't think it defaults to dev@... it just doesn't get used at all.

The discussions feature isn't enabled, so that setting is pointless to set... it's just using up space. I'd drop that to minimize the file and reduce confusion of what effect is it having.

```suggestion
```